Content.Server/GameObjects/Components/Arcade/BlockGameArcadeComponent.cs

```diff
@@ -124,18 +141,18 @@ private void UserInterfaceOnOnReceiveMessage(ServerBoundUserInterfaceMessage obj
             if (!(obj.Message is BlockGameMessages.BlockGamePlayerActionMessage message)) return;
```

Comment:
You can use a switch case here instead of doing if checks

Reply:
not really, since i want logic to run between the two checks